src/benchmark/listeners.ts

```diff
@@ -0,0 +1,97 @@
+import fibonacci from './cases/fibonacci';
+import newtonsMethod from './cases/newtons_method';
+import primeCounter from './cases/prime_counter';
+import quickSort from './cases/quicksort';
+import sumDoubles from './cases/sum_doubles';
+import sumIntegers from './cases/sum_integers';
+import Measurement from './Measurement';
+
+const algorithms = new Map<string, () => void>([
+    ['Fibonacci', fibonacci],
+    ['Newtons Method', newtonsMethod],
+    ['Prime Counter', primeCounter],
+    ['Quick Sort', quickSort],
+    ['Sum Doubles', sumDoubles],
+    ['Sum Integers', sumIntegers],
+]);
+
+function sum(value1: number, value2: number): number {
+    return value1 + value2;
+}
+
+function mean(values: number[]): number {
+    return values.reduce(sum, 0) / values.length;
+}
+
+function median(values: number[]): number {
```

Comment:
Is the median calculation still necessary? Maybe the variance is sufficient.

Reply:
I would just leave it there. It doesn't hurt to have more data.